OpenRA.Mods.Cnc/Traits/Infiltration/InfiltrateForCash.cs

```diff
@@ -32,6 +32,8 @@ class InfiltrateForCashInfo : ITraitInfo
 		[Desc("Sound the victim will hear when they get robbed.")]
 		public readonly string Notification = null;
 
+		public readonly bool ShowTicks = true;
```

Comment:
Can you please add `[Desc("Whether to show the cash tick indicators rising from the actor.")]`? (copied from `GivesCashOnCapture`)